src/ui/messages.ts

```diff
@@ -6,7 +6,7 @@ export namespace Messages {
     export const CompilationSucceeded = '$(book) Resolved (not verified)';
     export const Verifying = '$(sync~spin) Verifying';
     export const VerificationSucceeded = '$(thumbsup) Verification Succeeded';
-    export const VerificationFailed = '$(thumbsdown) Verification Failed';
+    export const VerificationFailed = '$(thumbsdown) Failed to verify';
```

Comment:
The intended change here is to also display how many proofs failed to verify, so instead of "Verification Failed" it says "Failed to verify X proofs". There's no real need for this change but I thought it would be a nice extra bit of information since that is available now.

"Could not prove X proofs" also seems fine to me.

Reply:
`Could not prove X declarations` looks good to me, thanks